packages/engine-server/src/doctor/service.ts

```diff
@@ -42,52 +44,63 @@ export class DoctorService {
     this.L = createLogger("DoctorService");
   }
 
-  getWildLinkDestinations(notes: NoteProps[], engine: DEngineClient) {
+  findWildLinks(note: NoteProps, notes: NoteProps[], engine: DEngineClient) {
     const { wsRoot, vaults } = engine;
+    const links = note.links;
+    if (_.isEmpty(links)) {
+      return [];
+    }
+
+    const out = _.filter(links, (link) => {
+      if (link.type !== "wiki") {
+        return false;
+      }
+
+      const hasVaultPrefix = LinkUtils.hasVaultPrefix(link);
+      let vaultPrefix: DVault | undefined;
```

Comment:
can we rename this since its not a vaultPrefix but actually a vault?

Reply:
good catch. fixed.